docs/versions/1.0.0.md

```diff
@@ -148,32 +158,36 @@ Requirements
    in [ISO 8601]{:target=_blank} format, including the time zone and a millisecond precision time part. (c) The values
    in the `Created` timestamps SHOULD reflect the correct order of the versions.
 
-3. The `bag-info.txt` file MAY contain at most one element called `Is-Version-Of` with a
+3. The `bag-info.txt` file MAY contain exactly one element called `Data-Station-User-Account`. If provided, it MUST
```

Comment:
After thinking it through my conclusion is that we should introduce `On-Behalf-Of` only when we are really going to use it. `Data-Station-User-Account` seems the best solution to pass the depositor ID to the validator. The SWORD 2 service can add it / check that it is equal to the authenticated user the client uses.

Reply:
i agree